project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 
 resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
I was updating based on the version suggestion here: http://www.scala-sbt.org/sbt-pgp/